### PR TITLE
the grammar is 'be used for'

### PR DESCRIPTION
**Pull request by liangwenguo**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go [hunk 1]
```diff
@@ -159,7 +159,7 @@ type Etcd struct {
 	// ExtraArgs are extra arguments provided to the etcd binary
 	// when run inside a static pod.
 	ExtraArgs map[string]string `json:"extraArgs,omitempty"`
-	// Image specifies which container image to use for running etcd.
```
Comment (dixudx) [inline cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go]: I think the old one is okay.

**Review by dixudx — CHANGES_REQUESTED** [review]

File: cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go (hunk 1 above)
Comment (hzxuzhonghu) [reply cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go]: agree

Comment (timothysc) [reply cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go]: agree closing this issue.

**Pull request closed without merging**